README.md

```diff
@@ -41,6 +42,8 @@ flags which include specified substring in either in the name, or description or
 path.
 ```
 
+The `enable_socket` flag defaults to `true` on Linux and `false` on MacOS.
```

Comment:
Fixed.

The file location on my machine is reported when the binary is executed, so I just had it copy-pasted - removed it altogether, since it does not offer any particularly useful information. The key information is in the flag set itself.